nala/utils/global.setup.js

```diff
@@ -60,8 +65,12 @@ async function getGitHubMiloLibsBranchLiveUrl() {
     console.info('PR Branch live url : ', prBranchLiveUrl);
     console.info('Milo Libs : ', miloLibs);
   } catch (err) {
-    console.error(`Error => Error in setting PR Branch test URL : ${prBranchLiveUrl}`);
-    console.info(`Note: PR branch test url  ${prBranchLiveUrl} is not valid, Exiting test execution.`);
+    console.error(
+      `Error => Error in setting PR Branch test URL : ${prBranchLiveUrl}`,
```

Comment:
why do we need the additional `,` here? Same for some of the other console messages.

Reply:
@narcis-radu : The additional comma is required to adhere **Eslint** `comma-dangle`, rule